examples/coap/src/bin/coapclient.rs

```diff
@@ -60,16 +60,22 @@ fn client_handshake() -> Result<(), EDHOCError> {
     let initiator = initiator.verify_message_2(valid_cred_r)?;
 
     let mut msg_3 = Vec::from(c_r.as_cbor());
-    let (mut initiator, message_3, prk_out) =
+    let (mut initiator, message_3, i_prk_out, i_prk_out_exporter) =
```

Comment:
Any reason to add `i_` to the prk_out variable? (if yes then please also update the coaphandler example by adding the prefix `r_`; if not, remove the prefixes here and in coapserver.)